content/articles/getting-started-clickfunnels.md

```diff
@@ -70,8 +64,7 @@ Welcome to [DNSimple](/articles/dnsimple-services/)! We're here to make your tra
 
 ## Step 3: Accepting and setting up the domain in DNSimple
 
-<div class="section-steps" markdown="1">
-1. [Log into DNSimple](https://dnsimple.com/login). You'll see a <label>Get Started</label> screen. Select <label>Skip this and go to the dashboard</label>, found under the three boxes of options. We'll go over these other options in the [section below](#other-options-when-getting-started-with-a-new-account).
+1. [Log in to DNSimple](https://dnsimple.com/login). You'll see a <label>Get Started</label> screen. Select <label>Skip this and go to the dashboard</label>, found under the three boxes of options. We'll go over these other options in the [section below](#other-options-when-getting-started-with-a-new-account).
 1. Welcome to the [Dashboard](https://dnsimple.com/dashboard) – the homepage of your DNSimple account. There's a box to the right that says <label>Alerts</label>. Click <label>View</label> to see your pending domain push and accept it.
```

Comment:
Is this still true or does it show up with the other domain alerts at the top of the page now?

Reply:
You are correct @itsalyse there is no longer a box with alerts. It will show at the top with the other notifications.
<img width="1185" alt="Screenshot 2024-06-26 at 15 26 01" src="https://github.com/dnsimple/dnsimple-support/assets/110252982/121ab1c3-2460-4948-beb4-35f14d3da705">